### ISSUE TITLE
TV Remote not displaying

### ISSUE THREAD

**Issue by captaincat25** [kiwi-cam/homebridge-broadlink-rm#220]
I have edited the config.json and entered all the information for my TV/Home Theatre remote. I followed all the instructions found in the documentation but there is no display for the remote. I checked both the home app and the remote app on my iPhone. I are able to use the Broadlink fine for things like on-off functions. The problem appears to be specific to "tv". Any suggestions? Below is the section of my config for this accessory.

"name": "Onkyo",
            "type": "tv",
            "data": {
                "on": "26006800000124951213113911150f15103a1015103911391213121312371238121411391039121312381138131311381313111411391138121410141238111412381139111411141100050c0001254c11000c420001274a12000c420001254b13000c420001254b12000d050000000000000000000000000000",
                "off": "26006800000124951213113911150f15103a1015103911391213121312371238121411391039121312381138131311381313111411391138121410141238111412381139111411141100050c0001254c11000c420001274a12000c420001254b13000c420001254b12000d050000000000000000000000000000",
                "remote": {
                    "select": "260018010001259313121238121312131238111412381238113812131238123812131238123812131139113812381213123812131213123812131114111412381213113813371312130004e60001274a11000c430001274a11000c430001264a12000c430001264a12000c420001274a12000c430001254b12000c420001264b12000c430001264b11000c430001274a12000c430001264b12000c430001264b11000c430001264b12000c430001264b11000c430001274a12000c440001254b12000c430001264b11000c430001274a12000c430001264b11000c430001264b12000c430001264b12000c430001264a13000c430001264a12000c430001264a12000c430001274a11000c440001264b12000c430001254b12000d050000000000000000000000000000",
                    "arrowUp": "260020010001269213121337131213121337131213371336133714111436133713121139133713121213133614111411141114111411143612381312113911381436143614361312130004e70001289114121337131213121337131114361436143614111436143613121337133713121312133713121312131213121312133713371312133713371337133713361411140004e70001279214111436131214111436131213371337193113121337133713121139133612131411143612131213141114111411143612381411143614361436133711391312130004e6000128921411143614111411143614111436123813371312133713371312113913361412131213361213141114111411141114361436141114361436133713371337131213000d05000000000000",
                    "arrowDown": "260020010001249413121138141112131238121312381238123811141139113812141138123812131238123812131213121312131213123811141114113911381238123812381213110004e80001269413131138121312131238121312381238123812131238123811141139113911141139113812131213121312131213123812131213123812381238123813371312110004e80001269511131238131213170e38121212381238123812131238123812131238123811141238123811141114111411141114123811141114113813371238123812381213120004e9000127921411133712131213123813121337133713371312133713371114113913361213133712381411121314111213121313371213121312381238123813371337121312000d05000000000000",
                    "arrowLeft": "260068010001249412131138121312131238131213371238123811141138123812131238133712131213121312381213111411141114113813371337121312381139123812371312120004e70001279412131139111412131138121411381238133712131238123812131238123811141213111412381114111412131114113912371337121313371337123812381213110004e90001259413121337121314111238121312381238123811141337123712141138133714111411141112381411121313121312143613371337111413371238113812381312120004e70001289313121337131213121336141213361436143614111436143613121337133713121114131213371312131213121114133614361436141114361436143613371312130004e6000128921411143614111411143613121337133711391312133713361313113814361411141114111436121312131411141114361337133713121337133713361436141114000d050000000000000000000000000000",
                    "arrowRight": "2600f8010001249412131238121312131337121312381139133613131237133712131238133712131238121312381114121312131213123811141139111411381238133713371312120004e80001279312131337121313121238121312381139123812131139113912131138123812131337131213371312131213121312123813121238121312381337123811391213120004e70001279313121238121313121238121313371238133712131238123811141238123812131139111411381313111411141213123712131238131212381238123813371213120004e700012794121311391213111411391114113912381138131213371337121313371337131213371312123812131213121312131337121312381213123a1038123812381114120004e70001269512131139111412131238121312371238123813121337133713121238133712131238121312381213121312131213123811141238111411391238113911381213120004e80001269412131238131213121238121312381337123812131238123812131139113911141238111412381213111412131114113911141238111412381138123812381213130004e7000127931312123812131312123813121238123812381312123812381213123812381114113912131139111412131213121311391213113912131139123812381138121412000d050000000000000000000000000000",
                    "playPause": "2600780000012593121311391114121312371312123812381312133712131213121311141114121312381114113911381312121313121238131213371213121312381238113911141200057b0001274a12000c430001264b12000c440001264b13000c430001254b13000c430001264a12000c440001264b12000d050000000000000000000000000000"
                },
                "volume": {
                    "up": "2600b0010001259312131238121312131238121312371337123811141238123811141138123812131213123812131213121311141114111412381213123712381535123812381238120004e60001269511141237121313121337121312381238123812131139123812131139123712131213123813121312121313121213121312381312123911381238113911391138130004e60001279313121238131212131238131212381238113911141139123811141237133713121213123812131213121312131213121312381213123811391139123811381238130004e60001289213121337111412131139111412381238113912141137133712131238123813121312123812131213121312131213121312381114123813371238123712381238130004e60001269412131238121312131238121312381139113911141139113812141138123813121312123812141113121312131213121313371213123811391139123811391237120004e8000126941114123811141213113813121337123812381312133712381213113911391213111412381213121311141114111411141238121311381337123812381238123811000d05000000000000",
                    "down": "2600f8010001249412131139121312131138131212381238123812131238123714111238133711141238113911141113131214111312121312131312133711391139113812381238110004e80001269411141139121311141138121312381238123812131238123811141139113911141138133712131213131212131213121313121213123812381139113911391138120004e70001269413121238131212131238121312381139123811141138123812131337123812131238123812131214111312131114111411141114123811391138123812381238130004e60001269412131238121312131238111412381238113911141237123814111238133712131238123811141114111411141114121311141213113911391138133712381337120004e70001269413121337121312131139121311391238123812131138133712131337133712131238113912131213121311141114111411141114123811381238133712381337120004e80001259412131238121312131238121311391139113912131139123713121238133712131238123812131213121312131114111412131114113911391238123713371337130004e6000127931312133713121213123a10131238123811391213123811391114113813371213133712381312131213121213131213121213121313371238123811391238113812000d050000000000000000000000000000"
                },
                "inputs": [
                    {
                        "name": "Apple TV",
                        "type": "hdmi",
                        "data": "2600900000012494111411381213121312381213123812381114111411391138121411381238121312131213123812381114111411141139113911391113121312381238113911141100050c0001264b11000c420001264b13000c410001264b13000c400001284a13000c400001264b13000c410001264b13000c400001274a12000c420001264b12000c420001264b13000d05000000000000"
                    },
                    {
                        "name": "FM Radio",
                        "type": "tuner",
                        "data": "260008010001269213121238131213121337131213361436143613121337133713121336143614111411143613371312131213121337131213361411141114361337133713121337130004e50001284913000c400001284913000c410001254b13000c410001274a13000c400001284914000c400001284913000c400001284914000c400001284914000c410001274914000c400001284914000c420001274913000c410001284913000c420001254b13000c410001284914000c410001264b11000c430001274a12000c430001264b12000c430001274a12000c430001264b11000c430001264b11000c430001264b12000c430001254b12000c430001264a12000c430001264b13000d050000000000000000000000000000"
                    }
                ]
            }

**Comment by masterfordev** [comment]
You added the accesory on Home app too?

**Comment by captaincat25** [comment]
I just tried adding it through the app. There's an Onkyo device listed (which is the brand name of my home theatre). When I tried to add it though t asks for an 8 digit code. I checked the Homebridge log and there's nothing generated for me to enter. I'm not sure what to do now.

Edit: I tried entering "00000000" and the code seems to be correct, however I got this error: "The accessory is not ready to be added to Home. Try powering it off and on, then try again. If you're still having trouble, refer to the manufacturer's guide".

I tried restarting Homebridge and I still get the same error.

Edit 2: I entered the code use for Homebridge and it now seems to be working properly!  I will close this issue.
